Rakefile

```diff
@@ -70,6 +68,26 @@ namespace :triannon do
   desc 'run test rails console w triannon but no jetty'
   task :console => :console_no_jetty
 
+  desc 'update triannon gems and engine cart app'
+  task :update_app do
```

Comment:
This is a convenience task that I find useful and would rather have a rake task for it than type it everytime.  The additional commentary in the rake task will help to remind me of why it is done the way it is done.